scripts/set-env.sh

```diff
@@ -0,0 +1,7 @@
+export PROJECT=$(gcloud config get-value project)
+export ZONE=us-central1-a
```

Comment:
Might consider changing zone to also use the gcloud command
`gcloud config get-value compute/zone`

Reply:
updated,